megatron/model/rwkv/v6/rwkv.py

```diff
@@ -7,7 +7,17 @@
 import torch.nn as nn
 from torch.nn import functional as F
 from torch.utils.cpp_extension import load
-
+from megatron import mpu
+from megatron.mpu import gather_from_model_parallel_region, reduce_from_model_parallel_region, scatter_to_model_parallel_region
+try:
+    from fla.ops.rwkv6 import chunk_rwkv6
+    import einops
+except ModuleNotFoundError:
+    print(
+        "Unable to import RWKV FLA kernels. Install them from our requirements/requirements-rwkv.txt, \
+    or directly from https://github.com/sustcsonglin/flash-linear-attention.git, or use CUDA kernels."
```

Comment:
This last point `"or use CUDA kernels"` is confusing. Can you add a `"by doing xyz"` so that users know what you mean?

Reply:
reminder^